### ISSUE TITLE
Intelx.io

### ISSUE THREAD

**Issue by cam-barts** [lanmaster53/recon-ng-marketplace#80]
**Is the feature request related to a tool? Please describe.**  
From the [site](https://intelx.io/):

```
Intelligence X allows you to perform a search for these selector types:
- Email address
- Domain, including wildcards like *.example.com
- URL
- IP, CIDR. Both IPv4 and IPv6 are fully supported.
- Phone Number
- Bitcoin address
- MAC address
- IPFS Hash
- Credit Card Number
- Social Security Number
- IBAN (International Bank Account Number)
... and other internal ones like UUID, Storage ID, Simhash
```
**Does the tool have a web API?**  

No it doesn't look like it, but it does have a decently cheap pro tier

**Describe the expected module functionality.**  
- Domain -> Creds
- Contacts -> Creds
- Domain -> Leaks
- Contact -> Leaks

**Comment by Kleissner** [comment]
We do have an API. The Software Development Kit is here: https://github.com/IntelligenceX/SDK

**Comment by MrJester** [comment]
@Kleissner Would it be possible to get a trial license of a premium account that I could use to build out these modules? Thanks.

**Comment by Kleissner** [comment]
@MrJester of course! Just upgraded your account. It's by default for 2 weeks but we can extend it if necessary
